src/explore-education-statistics-admin/src/components/form/FormThemeSelect.tsx

```diff
@@ -0,0 +1,86 @@
+import { Theme } from '@admin/services/themeService';
```

Comment:
following comment in `src/explore-education-statistics-admin/src/pages/publication/components/PublicationDetailsForm.tsx`

We can remove this file I believe.

Reply:
On second look, for this component we could optionally keep this one, as it's also used in `PublicationsTab.tsx` - However we could also just use FormSelect as it contains little logic - I'll defer to you on this one